isa-bel/react-native-workshop
# Feat - Add Filters for location and department

#### What does this PR do?

Adds filters for location and department for the questions.

#### Where should the reviewer start?

In `src/utils/navHelpers.js`, `src/components/Home/Home.js`, and ` src/components/ActionsRow/ActionsRow.js`.

#### How should this be manually tested?
1. Load the app.
    ```bash
    react-native run-ios
    ```
2. On any questions, click the department name.
3. Verify that the displayed list has questions of only that department.
4. Go back to the first screen.
5. On any questions, click the location.
6. Verify that the displayed list has questions of only that location.
7. Go back to the first screen.
8. On any questions, click the question text.
9. Verify that the question details are still displayed.

#### Any background context you want to provide?



#### What are the relevant tickets?



N/A

#### Checklist
